### PR TITLE
fixed mock script bug caused by change in xCitadelLocker

### PR DESCRIPTION
**Pull request by aj-maz**

### CODE REVIEW

File: scripts/deploy-mock.js [hunk 1]
```diff
@@ -251,7 +251,7 @@ async function main() {
   if (!fs.existsSync(scriptsDirectory)) {
     fs.mkdirSync(scriptsDirectory);
   }
-  fs.unlinkSync(path.join(scriptsDirectory, "testnet-addresses.json"))
+  fs.unlinkSync(path.join(scriptsDirectory, "testnet-addresses.json"));
```
Comment (mrbasado) [inline scripts/deploy-mock.js:254]: this won't create the file if it doesn't exsist

File: scripts/deploy-mock.js [hunk 2]
```diff
@@ -150,7 +150,7 @@ async function main() {
   /// =======  xCitadelLocker
   await xCitadelLocker
     .connect(governance)
-    .initialize(address(xCitadel), "Vote Locked xCitadel", "vlCTDL");
+    .initialize(address(xCitadel), address(gac), "Vote Locked xCitadel", "vlCTDL");
```
Comment (mrbasado) [inline scripts/deploy-mock.js:153]: this needs to be added in deploy-local as well unless its already fixed there

**Comment by mrbasado** [comment]
For the knightrounding round script could you add signers[0] as a guest? also would be good to sneak in the chainId 1337 change if you could

File: scripts/deploy-mock.js (hunk 2 above)
Comment (aj-maz) [reply scripts/deploy-mock.js:153]: It was making some other problems there, I will do complete cleanup of scripts later today

**Pull request merged**